packages/docs/docs/guides/architecture.mdx

```diff
@@ -160,28 +188,17 @@ sequenceDiagram
     end
 ```
 
-### What's the downside of the new serialization boundary?
-
-When AI.JSX and the UI are both on the client, you can do things like:
-
-```tsx
-const dataPromise = getMyPromise();
+**Pros:**
 
-<div>
-  <AI.jsx>
-    <ChatCompletion temperature={1}>
-      // highlight-next-line
-      <UserMessage>Write me a poem about {dataPromise}</UserMessage>
-    </ChatCompletion>
-  </AI.jsx>
-</div>;
-```
+- **Flexibility** -> Your AI.JSX logic can be access from multiple clients (web app, native app, etc.).
```

Comment:
I like having the keyword (e.g. "flexibility") at the start of each section. This is an improvement over what I had.